test/integration/daemonset/daemonset_test.go

```diff
@@ -99,8 +92,15 @@ func setupScheduler(
 	}
 
 	eventBroadcaster.StartRecordingToSink(ctx.Done())
-
 	go sched.Run(ctx)
```

Comment:
this will run without the informers started, is that something we should be worried?
it seems all the Run and Start are done later on the tests bodys

Reply:
so, basically we are moving the 

> 		// Start Scheduler
>		setupScheduler(ctx, t, clientset, informers)

to `setup` , but it seems that the informers weren't started before this change , :shrug: technically should be the same